src/server/pps/cmds/cmds.go

```diff
@@ -288,7 +290,7 @@ All jobs created by a pipeline will create commits in the pipeline's repo.
 		Use:   "run-pipeline pipeline-name [-f job.json]",
 		Short: "Run a pipeline once.",
 		Long:  fmt.Sprintf("Run a pipeline once, optionally overriding some pipeline options by providing a spec.  The spec looks like this:\n%s", exampleRunPipelineSpec),
```

Comment:
I think it would be good to include the extend pipeline spec docs that we have on GH in this help string.